lib/Image/Image.js

```diff
@@ -172,12 +183,22 @@ module.exports = kind(
 		draggable: 'false'
 	},
 
+	/**
+	* @private
+	*/
 	handlers: {
 		onload: 'handleLoad',
 		onerror: 'handleError'
 	},
 
 	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateSource', path: ['_src', 'placeholder']}
```

Comment:
placeholder is to be changed only if there is no source, I guess. If source is already loaded and we want to change just the place holder calling 'updateSource' doesn't seem a good thing in UI perspective.